### ISSUE TITLE
Undefined variable g:nortia_bat_light_theme

### ISSUE THREAD

**Issue by jedahan** [alaric/nortia.nvim#3]
When trying out this theme, I get

```vim
Error executing vim.schedule lua callback: [string ":lua"]:22: Vim(let):E121: Undefined variable: g:nortia_bat_light_theme 
```

**Comment by jedahan** [comment]
added the following and the error went away, maybe should be in the readme?

```lua
vim.g.nortia_bat_light_theme = true
```

**Comment by alaric** [comment]
Thanks for trying out the theme! This variable should have been optional but I'm more used to Lua than vimscript semantics and should have used `get()`. I've now fixed this in c2ec464e0f9dd6bf0b9effe25c043ce4b8fa5373 so you can remove the workaround if you're still using it.
